clients/drcachesim/common/options.cpp

```diff
@@ -854,5 +855,10 @@ droption_t<std::string>
                          "Path with stored as-traced schedule for replay.");
 #endif
 
+// Schedule_stats options.
+droption_t<uint64_t> op_print_every(DROPTION_SCOPE_ALL, "print_every", 5000,
```

Comment:
op_schedule_stats_print_every would be better.

Reply:
Done.